src/Symfony/Component/HttpKernel/DataCollector/ConfigDataCollector.php

```diff
@@ -288,4 +285,124 @@ private function determineSymfonyState()
 
         return $versionState;
     }
+
+    /**
+     * Gets the names, paths and installed versions of the bundles enabled in
+     * the application.
+     *
+     * @return array
+     */
+    public function getBundleData()
+    {
+        $bundles = array();
+        $installedPackages = $this->getInstalledPackages();
+        $enabledBundles = $this->getEnabledBundles();
+
+        foreach ($this->getEnabledBundles() as $bundleName => $bundlePath) {
+            foreach ($installedPackages as $packageName => $packageVersion) {
+                if (strpos($bundlePath.DIRECTORY_SEPARATOR, DIRECTORY_SEPARATOR.'vendor'.DIRECTORY_SEPARATOR.$packageName.DIRECTORY_SEPARATOR)) {
+                    $bundleVersion = $packageVersion;
+
+                    break;
+                } elseif (strpos($bundlePath, 'symfony'.DIRECTORY_SEPARATOR.'symfony'.DIRECTORY_SEPARATOR.'src'.DIRECTORY_SEPARATOR.'Symfony'.DIRECTORY_SEPARATOR.'Bundle')) {
+                    // this is a built-in Symfony bundle; its version is the same as Symfony
+                    $bundleVersion = Kernel::VERSION;
+
+                    break;
+                }
+
+                $bundleVersion = null;
+            }
+
+            $bundles[$bundleName] = array('name' => $bundleName, 'path' => $bundlePath, 'version' => $bundleVersion);
+        }
+
+        ksort($bundles);
+
+        return $bundles;
+    }
+
+    /**
+     * Finds the path of the 'composer.lock' file starting from the current
+     * kernel dir and going up recursively up until the project root dir.
+     *
+     * @return string
+     */
+    private function findComposerLockPath()
+    {
+        $path = $this->kernel->getRootDir();
+        $projectRootDirectory = $this->findProjectRootDirectory();
+
+        while ($path !== $projectRootDirectory) {
+            if (file_exists($composerLockPath = $path.'/composer.lock')) {
```

Comment:
Don't forget to use `DIRECTORY_SEPARATOR` here as well (just for extra safety :) ) .